Maple2.Server.Core/proto/world/world.proto

```diff
@@ -572,3 +575,12 @@ message DeathInfo {
   int32 ms_remaining = 2;
   int64 stop_time = 3;
 }
+
+message LockRequest {
+  int64 accountId = 1;
+}
+
+message LockResponse {
+  bool success = 1;
```

Comment:
this feels redundant, no? if error is empty then that equals success, I would assume.